src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    private String taskName;
+    private boolean isDone;
+
+    /**
+     * Constructor
+     * 
+     * @param name task name
+     * @return new task instance
+     */
+    public Task(String name) {
+        this.taskName = name;
+        this.isDone = false;
```

Comment:
Try to reduce the use of this. Use of this is discouraged, potentially to avoid unnecessary noise in the code. Try to avoid using this except when it is shadowed by a method or constructor parameter.